client/src/components/SearchableDropdownField/SearchableDropdownField.js

```diff
@@ -92,11 +92,9 @@ const SearchableDropdownField = ({
       setHasChanges(true);
       setJustChanged(true);
     }
-    if (multi) {
-      onChange(val);
-    } else {
-      onChange(val, { value: val.value });
-    }
+    // The first param is for handleChange() SearchableDropdownFieldEntwine.js
+    // The second param is for onChange in FormBuilder::buildComponent()
+    onChange(val, { value: val });
```

Comment:
This change has resulted in a regression of https://github.com/silverstripe/silverstripe-admin/issues/1733
Probably still needs to be `val.value`, at least for single fields. Please read the comments in the issue and the PR 'cause I recall there was discussion about the difference between single and multi fields, as well.